debugger/src/tpage.c

```diff
@@ -265,11 +264,9 @@ void tpage_pack_widgets(gboolean tabbed)
 
 	if (oldroot)
 	{
-		int i = 0;
-		while (widgets[i])
+		for (int i = 0; widgets[i]; i++)
 		{
 			g_object_unref(*widgets[i]);
```

Comment:
Is cppcheck happy with that?  Actually I'm still confused on what it didn't like before, so maybe it's happy. It's correct anyway so whichever way's OK with me.

Reply:
It is.
Only `gtk_container_remove(GTK_CONTAINER(gtk_widget_get_parent(*widgets[i])), *widgets[i]);` was problematic, I'm not sure why only here.

Reply:
Yeah I was just testing manually building cppcheck 2.2 :)